File: src/main/java/dqualizer/research/dqapi/repositories/dam/DomainArchitectureMappingRepository.java
package dqualizer.research.dqapi.repositories.dam;

import io.github.dqualizer.dqlang.types.dam.DomainArchitectureMapping;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DomainArchitectureMappingRepository extends MongoRepository<DomainArchitectureMapping, String> {
}

File: src/main/kotlin/io/github/dqualizer/dqapi/repositories/dam/DomainArchitectureMappingRepository.kt
package io.github.dqualizer.dqapi.repositories.dam

import io.github.dqualizer.dqlang.data.DAMMongoRepository
import org.springframework.context.annotation.Primary
import org.springframework.stereotype.Repository

@Repository
@Primary
interface DomainArchitectureMappingRepository : DAMMongoRepository
